
diff --git a/wp-content/plugins/accessible-poetry/inc/toolbar.php b/wp-content/plugins/accessible-poetry/inc/toolbar.php
index e5de02c5..2920151a 100644
--- a/wp-content/plugins/accessible-poetry/inc/toolbar.php
+++ b/wp-content/plugins/accessible-poetry/inc/toolbar.php
@@ -20,7 +20,7 @@ function acwp_toolbar_component() {
         </button>
     </div>
     
-    <div id="acwp-toolbar" class="acwp-toolbar <?php echo $side . ' ' . $noanimation . ' ' . $style; ?>" aria-label="<?php _e('Accessibility Toolbar Toggle View', 'acwp'); ?>">
+    <div id="acwp-toolbar" class="acwp-toolbar <?php echo $side . ' ' . $noanimation . ' ' . $style; ?>" role="navigation" aria-label="<?php _e('Accessibility Toolbar Toggle View', 'acwp'); ?>">
         <div id="acwp-toolbar-module">
             <?php acwp_toolbar_header(); ?>
             
diff --git a/wp-content/plugins/accessible-poetry/inc/notifications.php b/wp-content/plugins/accessible-poetry/inc/notifications.php
index 479f679c..72445233 100644
--- a/wp-content/plugins/accessible-poetry/inc/notifications.php
+++ b/wp-content/plugins/accessible-poetry/inc/notifications.php
@@ -75,7 +75,7 @@ function acc_userway_admin_script() {
                         // Set a timeout of 0.5 seconds before setting the cookie
                         setTimeout(function() {
                             // Set a cookie that the notice was dismissed
-                            document.cookie = "acc_userway_activation_notice_dismissed=true; expires=Fri, 31 Dec 9999 23:59:59 GMT; path=/";
+                            document.cookie = "acc_userway_activation_notice_dismissed=true; SameSite=Value; Secure; expires=Fri, 31 Dec 9999 23:59:59 GMT; path=/";
                         }, 500);
                     });
                 } else {
diff --git a/wp-content/plugins/accessible-poetry/inc/toolbar.php b/wp-content/plugins/accessible-poetry/inc/toolbar.php
index 2920151a..ad1f6b16 100644
--- a/wp-content/plugins/accessible-poetry/inc/toolbar.php
+++ b/wp-content/plugins/accessible-poetry/inc/toolbar.php
@@ -108,7 +108,7 @@ function acwp_toolbar_footer() {
             <?php if($sitemap_link && $sitemap_link != '') : ?>
             <li><a href="<?php echo $sitemap_link; ?>"><?php echo $sitemap_text;?></a></li>
             <?php endif; ?>
-            <li><?php _e('Powered with', 'acwp'); ?> <?php echo $icon;?> <?php _e('by', 'acwp');?> <a href="<?php echo $url;?>" target="_blank"><?php echo $cnr; ?></a></li>
+            <li><?php _e('Powered with', 'acwp'); ?> <?php echo $icon;?> <?php _e('by', 'acwp');?> <a href="<?php echo $url;?>" target="_blank" rel="noopener noreferrer nofollow"><?php echo $cnr; ?></a></li>
         </ul>
     </div>
     <?php

