diff --git a/wp-login.php b/wp-login.php
index 60d9c21f..a7617ab8 100644
--- a/wp-login.php
+++ b/wp-login.php
@@ -528,7 +528,14 @@ if ( defined( 'RELOCATE' ) && RELOCATE ) { // Move flag is set.
 
 // Set a cookie now to see if they are supported by the browser.
 $secure = ( 'https' === parse_url( wp_login_url(), PHP_URL_SCHEME ) );
-setcookie( TEST_COOKIE, 'WP Cookie check', 0, COOKIEPATH, COOKIE_DOMAIN, $secure, true );
+  setcookie(TEST_COOKIE, 'WP Cookie check', [
+    'expires' => 0,
+    'path' => COOKIEPATH,
+    'domain' => COOKIE_DOMAIN,
+    'secure' => $secure,
+    'httponly' => TRUE,
+    'samesite' => 'Lax',
+  ]);
 
 if ( SITECOOKIEPATH !== COOKIEPATH ) {
 	setcookie( TEST_COOKIE, 'WP Cookie check', 0, SITECOOKIEPATH, COOKIE_DOMAIN, $secure, true );
